app/router.js

```diff
@@ -8,6 +8,9 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('about');
   this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:title' });
```

Comment:
You would get: `/rentals/Beautiful%20house%20on%20the%20beach`.  The problem I ran into with `dasherize` is matching the title in the ember-cli-mirage mock.  We would have to add dasherize logic to the config file in order to match:
https://github.com/emberjs/super-rentals/pull/24/files#diff-04c022997e2228b6dc98593a3c6b7e87R58